packages/jaeger-ui/index.html

```diff
@@ -12,7 +12,7 @@
 
     <!-- NOTE: The document MUST have a <base> element. package.json#homepage is set to "." as part of resolving https://github.com/jaegertracing/jaeger-ui/issues/42 and therefore static assets are linked via relative URLs. This will break on many document URLs, e.g. /trace/abc, unless a valid base URL is provided. The base href defaults to "/" but the query-service can inject an override. -->
     <base href="/" data-inject-target="BASE_URL" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="/favicon.ico">
```

Comment:
does this work with `base_path != "/"` too?

Reply:
Looks like the favicon actually gets inlined via a `data:` URI. For other `link` tags such as the generated `link` tag injected for CSS chunks, they use an appropriate relative base URL (`./static`).